vms_activity/models/vms_activity.py

```diff
@@ -57,85 +57,74 @@ def _compute_total_hours(self):
                 sum_time += (end_date - start_date).total_seconds()/3600
             rec.total_hours = float("%.2f" % sum_time)
 
-    @api.multi
-    def start_end_activity_time(self, rec):
-        """ Toggle action to start or end an activity. """
-        act_in_process = self.end_activity_time(rec, throw_back_act=True)
-        if not act_in_process:
-            other_act = self.search([
-                ('id', '!=', rec.id),
-                ('responsible_id', '=', rec.responsible_id.id),
-                ('state', '=', 'process')], limit=1)
-            if other_act:
-                raise ValidationError(_('There is another task in process.'))
+    @staticmethod
```

Comment:
Why you change it to staticmethod?

Reply:
Because It marks that i don't need self in my method, and if i don't set as staticmethod it marks that need self, but let me change it again.

Reply:
Instead of using staticmethod you can use @api.model with self, but it won't ask for self.